mne/viz/topo.py

```diff
@@ -437,16 +446,16 @@ def _plot_timeseries_unified(bn, ch_idx, tmin, tmax, vmin, vmax, ylim, data,
     """Show multiple time series on topo using a single axes."""
     import matplotlib.pyplot as plt
     if not (ylim and not any(v is None for v in ylim)):
-        ylim = np.array([np.min(data), np.max(data)])
+        ylim = [min([np.min(d) for d in data]), max([np.max(d) for d in data])]
```

Comment:
For future ref `min` and `max` can handle iterators so you can just do `min(np.min(d) for d in data)` and get a tiny char and efficiency gain (no need to construct list via iteration, then iterate again)

Reply:
Good point.